dbcmetrics/dbcm_logic/dbcplannedmaintenance.py

```diff
@@ -0,0 +1,150 @@
+        self.thread.daemon = True
+        self.thread.start()
+        logging.info(f"Planned Maintenance Metrics Thread started. UTC Time: {datetime.datetime.utcnow()}")
+
+    def run(self):
+        while True:
+            if (time.time() - self.last_time_windows_loaded)/60 > self.LOADING_INTERVAL_MIN:
+                self.load_planned_maintenance_windows()
+                self.last_time_windows_loaded = time.time()
+            self.refresh_metrics()
+            sleep(self.METRICS_INTERVAL_SEC)
+
+
+    def load_planned_maintenance_windows(self):
+
+        # windows is a dict of all platforms e.g.: {"status.plattform1": [["03.11.2022 13:00","03.11.2022 12:00"]["03.11.2022 13:00","03.11.2022 12:00"]], "next.Platform": ...}
+        #                                            platform name          window_start_date      window_end_date   window_start_date  window_end_date       ...
+        
+        for platform in self.PLATFORMS:
+            try:
+                platform_name = platform['name']
+                logging.info(f"platform_name = {platform_name}, plattform_url = {platform['url']}")
+
+                url = platform['url'] + "/api/v1/schedules"
+
+                headers = {
+                    "accept": "application/json"
+                }
+
+                response = json.loads(requests.get(url, headers=headers).text)
+
+                platform_windows = []
+                # Several maintance_entry may exist in response['data'], default is empty List
+                for maintance_entry in response['data']:
+                    # load and parse data from response
+
+                    try:
+                        logging.info(f"Found Mainenance entry in {platform_name}, title: {maintance_entry['name']}, message: {maintance_entry['message']}")
+                    except Exception as e:
+                        # Triggered if message field is empty
+                        # It is not a requiered field, so code will continue for this entry
+                        logging.warning(f"Couldn't load maintenance entry: title or message on {platform_name} for entry: {maintance_entry}")
+                        logging.exception(f"\t {e}")
+
+
+                    try:
+                        platform_window_start = self.parse_string_to_datetime(maintance_entry['scheduled_at'])
+                    except Exception as e:
+                        # Triggered if field for start of platform maintenance window is empty
+                        logging.warning(f"Couldn't load maintenance entry: platform_window_start on {platform_name} for entry: {maintance_entry}")
+                        logging.exception(f"\t {e}")
+                        continue
+
+                    try:
+                        message_string: str = maintance_entry['message']
+                        stunde_result = re.findall(r"([\d]+)\s*h", message_string) # returns e.g.: 2, 10, 1 as String in a List
```

Comment:
The regex should be more specific. For example it also matches hier10 hallo

Reply:
> The regex should be more specific. For example it also matches hier10 hallo

Has now been improved